presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -301,7 +301,7 @@ public Void visitUnion(UnionNode node, Void context)
         @Override
         public Void visitRemoteSource(RemoteSourceNode node, Void context)
         {
-            printNode(node, "Exchange 1:N", NODE_COLORS.get(NodeType.EXCHANGE));
+            printNode(node, (node.getOrderingScheme().isPresent() ? "Merge" : "Exchange") + " 1:N", NODE_COLORS.get(NodeType.EXCHANGE));
```

Comment:
Should `visitExchange` and `visitSort` methods be updated to distinguish between partial and final sort as well as gather and merge-gather exchange?

Reply:
I'm not sure really. I think it's worth noting that data is also merged as part of exchange. I don't think we can easily distinguish between non-distributed `Sort` operator and distributed `Sort` operator though.